utils/req-res-log-validator.py

```diff
@@ -56,7 +56,9 @@
     "debug",
     "pfdebug",
     "lolwut",
-]
+    "sentinel|debug",
+    "sentinel|info-cache",
```

Comment:
to skip them first, they both lack the reply schema...
It's kind of hard for me to describe them, maybe someone can take a quick look

Reply:
i remove the info-cache one, and add another `sentinel|simulate-failure`

i can add some tests about `sentinel|simulate-failure` tomorrow or later

Reply:
i'ave asked @hwware to review your schemas and either help you fix them.
i'm guessing the tests will have a bigger gap to get them to do more than just coverage. please collaborate on that.